docs/Materials.md.html

```diff
@@ -108,6 +110,8 @@
 **clearCoatNormal**     | float3   |  [0..1]                  | Linear RGB, encodes a direction vector in tangent space
 **emissive**            | float4   |  rgb=[0..1], a=[-n..n]   | Alpha is the exposure compensation
 **postLightingColor**   | float4   |  [0..1]                  | Pre-multiplied linear RGB
+**transmission**        | float    |  [0..1]                  |
+**absorption**          | float3   |  [0..1]                  | Linear RGB, encodes how much of each component is absorbed
```

Comment:
Is it linear RGB? I don't think the absorption coefficients are an actual color, just absorption coefficients for each color channel

Reply:
right, but they are linear *and* color channels, right?